boards/waveshare/rp2040_zero/doc/index.rst

```diff
@@ -0,0 +1,151 @@
+.. _rp2040_zero:
+
+Waveshare RP2040-Zero
+#####################
```

Comment:
```suggestion
.. zephyr:board:: rp2040_zero
```

Reply:
I added the `full_name` field in the board.yaml.

Now it works correctly.